css/style.css

```diff
@@ -156,6 +156,78 @@ a {
   justify-items: center;
```

Comment:
- [ ] The popup follows the design for both desktop and mobile versions from the chosen template in Figma including the font, colors, images, and text, kindly use the same styling in your design

|Your design|Figma design|
|-|-|
|![Screenshot_20220720_062854](https://user-images.githubusercontent.com/25227022/179904182-a911e6cd-12d5-448a-b696-98712247d9d0.png)|![Screenshot_20220720_063258](https://user-images.githubusercontent.com/25227022/179904342-d1bf5178-26bc-4310-a9be-0ad96290c0e5.png)|